acceptancetests/jujupy/utility.py

```diff
@@ -128,6 +128,11 @@ def split_address_port(address_port):
     return address, port
 
 
+def get_unit_ipaddress(client, unit_name):
```

Comment:
We probably want to be explicit here and call this something like get_unit_public_ip

Reply:
Good point, I've changed that around.